internal/reflectunsafe/reflectunsafe.go

```diff
@@ -0,0 +1,12 @@
+package reflectunsafe
+
+import (
+	"reflect"
+	"unsafe"
+)
+
+func ValueAndTypeID(v any) (reflect.Value, uintptr) {
```

Comment:
Maybe we can make this util?

Reply:
I dont like putting much dirs in internal. Maybe we can put it inside of internal/bind. Wdym?

Reply:
that't fine.